Extensions/PrimitiveDrawing/shapepainterruntimeobject.js

```diff
@@ -5,18 +5,18 @@
 
 /**
  * @typedef {Object} RGBColor Represents a color in RGB Format
- * @property {number} r The Red portion of the color from 0 to 255
- * @property {number} g The Green portion of the color from 0 to 255
- * @property {number} b The Blue portion of the color from 0 to 255
+ * @property {number} r The Red component of the color, from 0 to 255.
+ * @property {number} g The Green component of the color, from 0 to 255.
+ * @property {number} b The Blue component of the color, from 0 to 255.
  */
 
 /**
- * @typedef {Object} ShapePainterObjectDataType Base parameters for gdjs.ShapePainterRuntimeObject
- * @property {RGBColor} fillColor The color as RGB representation of the inner part of the painted shape
- * @property {RGBColor} outlineColor The color as RGB representation of the outline of the painted shape
+ * @typedef {Object} ShapePainterObjectDataType Initial properties for a for {@link gdjs.ShapePainterRuntimeObject}.
```

Comment:
Please no period for consistency

Reply:
I do think we should include periods for parameters and properties description. After all they are sentences, so should be finished by a period. This is especially visible when there are two sentences (why finish the first by a period but not the second) or when a question mark is used (if using a question mark, why not putting a period).

That's bike shedding though, let's go for periods and it's really not a big deal if some are missing.